clib/src/main/scala/scala/scalanative/libc/string.scala

```diff
@@ -3,8 +3,10 @@ package libc
 
 import scalanative.unsafe._
 
-@extern
-object string {
+@extern object string extends string
+
+/** Currently following C99 */
```

Comment:
Better with the comment, but left wondering...

Could the Scaladoc comment be extended by a sentence
or two.

Why & how are these C99?   We compile using C11.
Certainly do not want to break users who compile with C99
but it seems like an odd restriction.

Reply:
Removed and put the C99 reference in the docs. We can follow C11 and add additional functions if we want but we would have to decide as a team.